packages/usa-tooltip/src/index.js

```diff
@@ -369,7 +369,10 @@ const tooltip = behavior(
       [TOOLTIP_TRIGGER](e) {
         const { trigger, body } = getTooltipElements(e.target);
 
-        showToolTip(body, trigger, trigger.dataset.position);
+        // Default to "top" if `data-position` not specified
+        const position = trigger.dataset.position || "top";
```

Comment:
This was supposed to be handled in  `:318`. We check for position in `setUpAttributes` and use the default. We should revisit this.

---


Allternatively, we could pass it as a default parameter in `showTooltip`:

```js
const showToolTip = (tooltipBody, tooltipTrigger, position="top") => {
```

Reply:
You're right, that's a better solution. Updated the PR. Thanks @mejiaj!